README.md

```diff
@@ -23,7 +25,7 @@ The Virgil SDK allows developers to get up and running with Virgil API quickly a
 - CMake 3.10+
 
 ### CMake
-Virgil SDK can be integrated using CMake in the different ways
+Virgil SDK can be integrated using CMake in two different ways
```

Comment:
There are more than two ways to integrate projects using CMake. I've just described two of them.